native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java

```diff
@@ -260,11 +260,25 @@ protected void onCreate(Bundle savedInstanceState) {
                         ViewGroup.LayoutParams.MATCH_PARENT);
         RelativeLayout frameLayout = new RelativeLayout(this);
         frameLayout.setLayoutParams(frameLayoutParams);
+        frameLayout.setClickable(true);
+        frameLayout.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                CocosEditBoxActivity.this.hide();
+            }
+        });
         setContentView(frameLayout);
 
         this.addItems(frameLayout);
 
-        Bundle extras = getIntent().getExtras();
+        Intent intent = getIntent();
+        Bundle extras = null;
+        if (null != intent) {
+            extras = intent.getExtras();
+        }
+        if(extras == null){
+            extras = new Bundle();
```

Comment:
Empty Bundle will return null for extras.getString("defaultValue"). Could `show` handle null ?

Reply:
No, Editbox::show can handle a default value, but not null.

Reply:
Added a default show when no information was given